src/frontend/styles/components/sidebar.pcss

```diff
@@ -1,5 +1,4 @@
 .docs-sidebar {
-  width: 100vw;
```

Comment:
why not just replace `100vw` with `100%`?

Reply:
There is no need to specify an additional width for this element on mobile devices:
  - block-level elements, by default, take up the entire width of the parent (100%)
  - [mobile-first](https://github.com/codex-team/codex.docs/blob/main/src/frontend/styles/vars.pcss#L284) approach prevents default styles from being overridden

Reply:
Ok, seems reasonable